samples/server/petstore/python-aiohttp/openapi_server/openapi/openapi.yaml

```diff
@@ -32,7 +32,7 @@ paths:
         required: true
         x-body-name: body
       responses:
-        405:
+        "405":
```

Comment:
These spec updates are unexpected in this pull review because we did not edit the spec file.
Can you please add the below sample generation files to https://github.com/OpenAPITools/openapi-generator/blob/master/bin/utils/ensure-up-to-date?
- python-aiohttp
- python-flask-python2
- python-flask

Reply:
Added to ensure-up-to-date.

These scripts are using petstore.yaml (original) but not the fake petstore spec (which has numeric enums)

Reply:
Can we add a model with numeric enums in petstore.yaml?
That lets us verify this fix in the repo and this PR.

Reply:
We can update the script to use the fake petstore spec instead (petstore.yaml shouldn't be updated)

When a generator gets mature, we will use fake petstore spec instead of petstore.yaml/json as the generator should be able to handle those edge cases in fake petstore spec.

I've not sure if these generators can handle the edge cases in fake petstore spec.